lib/broccoli/amd-shim.js

```diff
@@ -6,11 +6,14 @@ module.exports = function shimAmd(tree, nameMapping) {
     let name = nameMapping[relativePath];
     if (name) {
       return [
-        '(function(define){\n',
+        '(function(define, module){\n',
```

Comment:
@rwjblue Regarding shifting the first argument when it is a string: libraries may [define their own name and use it as first argument](https://github.com/jquery/jquery/blob/3.1.1/dist/jquery.js#L10180). This will be conflicting with our wrapping factory, because the result will be `define("jquery", "jquery", factory);`.